### PR TITLE
1.19.4-pre2 stuff

### PR DESCRIPTION
**Pull request by apple502j**
Tiny pre-release. Also fixed javadoc.

### CODE REVIEW

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 1]
```diff
@@ -300,3 +301,25 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
...
+		ARG 8 centerSliceHeight
+		ARG 9 u
+		ARG 10 v
+	METHOD method_49698 setScissor (Lnet/minecraft/class_8030;)V
+		ARG 0 rect
+	CLASS class_8214 ScissorStack
+		FIELD field_43099 stack Ljava/util/Deque;
+		METHOD method_49699 pop ()Lnet/minecraft/class_8030;
+		METHOD method_49700 push (Lnet/minecraft/class_8030;)Lnet/minecraft/class_8030;
+			ARG 1 rect
```
Comment (Juuxel) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping:325]: Hmm, is FocusedRect the correct name for this type now? I don't think there's anything particularly "focused" about scissored areas.

File: mappings/net/minecraft/data/server/tag/TagProvider.mapping [hunk 2]
```diff
@@ -37,7 +39,13 @@ CLASS net/minecraft/class_2474 net/minecraft/data/server/tag/TagProvider
 		ARG 2 info
 	METHOD method_49660 (Lnet/minecraft/class_2474$class_8210;Lnet/minecraft/class_2960;)Z
 		ARG 2 id
+	METHOD method_49661 (Ljava/lang/Void;)Lnet/minecraft/class_2474$class_8211;
+		ARG 1 void_
```
Comment (Juuxel) [inline mappings/net/minecraft/data/server/tag/TagProvider.mapping:43]:
```suggestion
		ARG 1 v
```
The underscore is a bit ugly, and I think we use single letters for some lambda params elsewhere

**Review by Juuxel — APPROVED** [review]

File: mappings/net/minecraft/data/server/tag/TagProvider.mapping (hunk 2 above)
Comment (apple502j) [reply mappings/net/minecraft/data/server/tag/TagProvider.mapping:43]: We can mass-rename later.

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping (hunk 1 above)
Comment (apple502j) [reply mappings/net/minecraft/client/gui/DrawableHelper.mapping:325]: Guess I should rename this, probably in another PR though.

**Pull request merged**